src/pretix/plugins/stripe/static/pretixplugins/stripe/pretix-stripe.js

```diff
@@ -196,28 +241,47 @@ $(function () {
         return;
 
     if ($("input[name=payment][value=stripe]").is(':checked') || $(".payment-redo-form").length) {
-          pretixstripe.load();
+        pretixstripe.load();
     } else {
         $("input[name=payment]").change(function () {
-            if ($(this).val() === 'stripe') {
+            if (['stripe', 'stripe_sepa_debit'].includes($(this).val())) {
                 pretixstripe.load();
             }
         })
     }
 
     $("#stripe_other_card").click(
         function (e) {
-            $("#stripe_payment_method_id").val("");
+            $("#stripe_card_payment_method_id").val("");
             $("#stripe-current-card").slideUp();
-            $("#stripe-elements").slideDown();
+            $("#stripe-card-elements").slideDown();
 
             e.preventDefault();
             return false;
         }
     );
 
     if ($("#stripe-current-card").length) {
-        $("#stripe-elements").hide();
+        $("#stripe-card-elements").hide();
+    }
+
+    $("#stripe_other_account").click(
```

Comment:
Should we include sepa_debit in the name of these IDs as well in case we have other stripe methods later that also use the word "account"?